packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -124,6 +124,9 @@ const DropdownMenuTrigger = React.forwardRef<DropdownMenuTriggerElement, Dropdow
               if (!context.open) event.preventDefault();
             }
           })}
+          onFocusCapture={(event)=>{
+            event.stopPropagation();  
+          }}
```

Comment:
We want to treat this like any other event handle and compose it with the consumer's `onFocusCapture` prop
```suggestion
          onFocusCapture={composeEventHandlers(props.onFocusCapture, (event)=>{
            event.stopPropagation();  
          })}
```